core/lib/config/src/testonly.rs

```diff
@@ -343,9 +343,10 @@ impl Distribution<configs::eth_sender::ProofLoadingMode> for EncodeDist {
 impl Distribution<configs::eth_sender::PubdataSendingMode> for EncodeDist {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> configs::eth_sender::PubdataSendingMode {
         type T = configs::eth_sender::PubdataSendingMode;
-        match rng.gen_range(0..2) {
+        match rng.gen_range(0..3) {
             0 => T::Calldata,
-            _ => T::Blobs,
+            1 => T::Blobs,
+            _ => T::Custom,
```

Comment:
Will that break config compatibility? If one doesn't specify `PubDataSendingMode` now - they'd have "Blobs" enabled as a default. After that change it's going to be "Custom" which will change the node behaviour without partner realising that.

Reply:
this one is testonly, but in general - it won't
the ones who are running the chain already have `ETH_SENDER_SENDER_PUBDATA_SENDING_MODE = "Blobs"` in their configs, so it won't affect them

Reply:
Ok good - resolving